content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration.rst

```diff
@@ -12,6 +12,7 @@ Setup and configuration
    setup_configuration/labels
    setup_configuration/bpost
    setup_configuration/dhl_credentials
+   setup_configuration/fedex
```

Comment:
hahah thank you for putting this doc in alphabetical order to match the others :blush: